### PR TITLE
work for #266 - event table enhancements

### PR DESCRIPTION
**Pull request by djtfmartin**
Includes:
* some minor fixes for sites table
*  i18n
* Enable month filter
* changes to tableConfig to allow "set as filter"

### CODE REVIEW

File: packages/react-components/src/entities/SiteSidebar/SiteSidebar.js [hunk 1]
```diff
@@ -96,17 +96,19 @@ export function SiteSidebar({
 
 export function TemporalDisplay({ year, month }) {
   if (year && month && month > 0) {
+
+    let monthFormatted = useIntl().formatMessage({ id: `enums.month.${month}` });
     return (
       <Eyebrow 
         style={{fontSize: '80%'}}
-        prefix={<>Details of activity in <FormattedMessage id={`enums.month.${month}`} defaultMessage={`${month}`} />, {year}</>}
+        prefix={<><FormattedMessage id="eventDetails.activityDetails" values={{ date: monthFormatted + ', ' + year }}  /></>}
```
Comment (MortenHofft) [inline packages/react-components/src/entities/SiteSidebar/SiteSidebar.js:103]: Using intl date formatting seems a more natural fit?

File: packages/react-components/src/entities/SiteSidebar/SiteSidebar.js [hunk 2]
```diff
@@ -57,7 +57,7 @@ export function SiteSidebar({
       <Col shrink={false} grow={false} css={css.detailDrawerContent({ theme })} >
         {isLoading && (
           <Col style={{ padding: '12px', paddingBottom: 50, overflow: 'auto' }} grow>
-            <h2>{siteID} - Loading site information...</h2>
+            <h2><FormattedMessage id={`eventDetails.site`} />: {siteID} - Loading site information...</h2>
```
Comment (MortenHofft) [inline packages/react-components/src/entities/SiteSidebar/SiteSidebar.js:60]: Loader not translated

File: packages/react-components/src/search/EventSearch/views/Table/index.js [hunk 3]
```diff
@@ -143,7 +191,7 @@ const defaultTableConfig = {
       trKey: 'filters.measurementOrFactTypes.name',
       value: {
         key: 'measurementOrFactTypes',
-        formatter: (value, item) => <>{(value || []).join(',')}</>
+        formatter: (value, item) => <>{(value || []).slice(0, 10).join(',')}</>
```
Comment (MortenHofft) [inline packages/react-components/src/search/EventSearch/views/Table/index.js:194]: tiny: space after comma?

File: packages/react-components/src/entities/SiteSidebar/SiteSidebar.js (hunk 1 above)
Comment (djtfmartin) [reply packages/react-components/src/entities/SiteSidebar/SiteSidebar.js]: changed to use  intl date formatting

**Pull request merged**